internal/env/env.go

```diff
@@ -0,0 +1,20 @@
+package env
+
+import (
+	"os"
+	"strconv"
+)
+
+// GetEnvBool Extracts a boolean from an env var. Falls back to the default
+// if the key is unset or not a valid boolean.
+func GetEnvBool(key string, defaultValue bool) bool {
```

Comment:
I’d name this `GetBool` since its already in the `env` package. This is covered in the “zen of go” or w/e that Go document is (on golang.org). Try not to repeat the package name in the function names.